docs/docs/0.5.1/policies/traffic-log.md

```diff
@@ -138,11 +118,111 @@ spec:
     # Forward the logs into the logging backend named `logstash`.
     backend: logstash
 ```
+:::
+::: tab "Universal"
+```yaml
+type: TrafficLog
+name: all-traffic
+mesh: default
+# This TrafficLog policy applies to all traffic in the Mesh.
+sources:
+  - match:
+      service: '*'
+destinations:
+  - match:
+      service: '*'
+# When `backend ` field is omitted, the logs will be forwarded into the `defaultBackend` of that Mesh.
+```
+
+```yaml
+type: TrafficLog
+name: backend-to-database-traffic
+mesh: default
+# this TrafficLog policy applies only to traffic from service `backend` to service `database`.
+sources:
+  - match:
+      service: backend
+destinations:
+  - match:
+      service: database
+conf:
+  # Forward the logs into the logging backend named `logstash`.
+  backend: logstash
+```
+:::
+::::
 
 ::: tip
 When `backend ` field of a `TrafficLog` policy is omitted, the logs will be forwarded into the `defaultBackend` of that `Mesh`.
 :::
 
+## Log aggregation and visualisation
+
+`Kuma` is providing a simple solution to aggregate the **logs of your containers** and the **access logs of your data-planes**.
+This solution is available for `kubernetes`. If you're using `kuma` in universal mode you can forward the access logs to 
+a collector (such as logstash) that can further transmit them into systems like Splunk, ELK and Datadog.
```

Comment:
Can we use Loki in Universal? Can we point the users to an installation guide. I found this one but maybe we can be a bit more specific and have our own instructions:
https://github.com/grafana/loki/blob/v1.5.0/docs/installation/README.md

Reply:
I'll work on some doc for Loki in universal tomorrow :)

Reply:
I just pushed some documentation for loki in universal. Tell me what you think about it.